### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by GeoXiaN**
Well, there it is.

### CODE REVIEW

**Comment by GeoXiaN** [comment]
I have no clue what is going on =)

File: languages/ru.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  "Year": "Год",
+  "Play Queue": "Проиграть очередь",
+  "Home": "Домой",
+  "Search": "Поиск",
+  "Now Playing": "В проигрыше",
+  "Playlists": "Плейлист",
+  "Uploads": "Загрузка на сайт",
+  "Login": "Логин",
+
+  "Nothing here yet...": "Тут нечего нет...=(",
```
Comment (purplesyringa) [inline languages/ru.json:17]: Тут ничего нет (it has difference like "nothing" and "no thing")

File: languages/ru.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  "Press the Upload button to get started.": "Нажмите кнопку Загрузки на сайт чтобы начать.",
```
Comment (purplesyringa) [inline languages/ru.json:18]: Нажмите кнопку Загрузить, чтобы начать

File: languages/ru.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  "Upload Songs": "Загрузить Песни",
+  "Upload More Songs": "Загрузить ещё больше песен!",
+  "No songs in queue.": "Нет песен в очереди.",
+  "No playlists. Press '+' to add.": "Плейлиста нет. Нажмите на '+' чтобы добавить.",
```
Comment (purplesyringa) [inline languages/ru.json:22]: Плейлист не выбран. Нажмите '+', чтобы его создать

File: languages/ru.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  "Nothing Playing": "Нечего не играет... пока",
```
Comment (purplesyringa) [inline languages/ru.json:23]: Ничего не играет - same. Or "Ничего не играет... пока" (nothing playing... yet)

File: languages/ru.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  "Songs": "Песни",
+  "Type a search query to begin": "Нужмите суда для начала поиска",
```
Comment (purplesyringa) [inline languages/ru.json:25]: "Введите запрос, чтобы начать"

File: languages/ru.json [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  "Search artists, albums, songs...": "Ищем через исполнителя, альбомы, песни...",
+  "Nothing found.": "Нечего не найдино.",
```
Comment (purplesyringa) [inline languages/ru.json:27]: Ничего не найдено - same nothing typo

File: languages/ru.json [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  "Loading...": "Загрузка...",
+  "seed": "Сид",
+  "seeds": "Сиды",
+
+  "Queue Artist": "В очередь по Исполнителю",
+  "Queue Album": "В очередь по Альбому",
+  "Queue Song": "В очередь по Песне",
+  "Play Artist": "Приграть Исполнителя",
```
Comment (purplesyringa) [inline languages/ru.json:35]: Пр**о**играть исполнителя - missed letter

File: languages/ru.json [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  "Play Album": "Приграть Альбом",
```
Comment (purplesyringa) [inline languages/ru.json:36]: Проиграть альбом - same (I also changed capitalization)

File: languages/ru.json [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+  "Play Song": "Приграть Песню",
```
Comment (purplesyringa) [inline languages/ru.json:37]: Проиграть песню

File: languages/ru.json [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  "Download Artist": "Загрузить Исполнителя",
+  "Download Album": "Загрузить Альбом",
+  "Download Song": "Загрузить Песню",
+  "Remove Artist": "Убрать Исполнителя",
+  "Remove Album": "Убрать Альбом",
+  "Remove Song": "Убрать Песню",
+  "Remove Download": "Убрать Загрузку",
+  "Clear Playlist": "Отчистить Плейлист",
+  "Edit Song": "Изменить Песню",
```
Comment (purplesyringa) [inline languages/ru.json:46]: Очистить - that's like "clean" (before) and "clear" (after)

File: languages/ru.json [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+  "Song Files That Already Exist": "У нас уже есть этот Музыкальный файл",
+  "Song Tags That Already Exist": "У нас уже есть этот Музыкальный Таг",
+  "Track Number": "Намер трека",
```
Comment (purplesyringa) [inline languages/ru.json:50]: Таг - Тэг (tag in English) - not sure which word is used more often, probably the latter

File: languages/ru.json [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+  "Title": "Имя",
```
Comment (purplesyringa) [inline languages/ru.json:51]: Название ("name" originally)

File: languages/ru.json [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+  "This song is part of a compilation album.": "Эта песня часть сборочного альбома.",
```
Comment (purplesyringa) [inline languages/ru.json:52]: Эта песня входит в сборочный альбом

File: languages/ru.json [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+  "Compilation Album": "Сборочный Альбом",
+  "Genres": "Жанры",
+  "Upload Art": "Загрузить Арт",
+  "Delete Art": "Удалить Арт",
+  "Merge Song": "Собрать песни",
```
Comment (purplesyringa) [inline languages/ru.json:57]: Объединить песни (join songs) or Склеить песни (glue songs). Собрать песни (originally) is "collect/gather songs".

**Comment by purplesyringa** [comment]
Fixed some typos. ~native Russian speaker

**Comment by anoadragon453** [comment]
@Imachug were you going to submit a patch?

**Comment by purplesyringa** [comment]
TBH I don't have much time now, so I thought you can copy-paste my changes.

**Comment by anoadragon453** [comment]
Sure, I just don't see where your changes are?

**Comment by purplesyringa** [comment]
Open `Files changed` tab, you'll see my comments.

**Comment by anoadragon453** [comment]
screenshot_2018-06-10_17-59-43

I don't though :/

**Comment by anoadragon453** [comment]
Are you doing a review? Perhaps you have to submit it first with the Review Changes button in the top right.

**Comment by GeoXiaN** [comment]
Oh, man... I already see some typos. Yeah, better fix it.

**Comment by ghost** [comment]
Maybe they aren't showing up because this is already merged?

**Comment by anoadragon453** [comment]
Possibly? @GeoXiaN is going to make another PR to fix stuff up.

**Comment by purplesyringa** [comment]
Sent review now.

File: languages/ru.json (hunk 14 above)
Comment (anoadragon453) [reply languages/ru.json:57]: This should be song singular. Is that what песни translates to?

**Comment by anoadragon453** [comment]
Apologies for the delay, a7eb2dfcfd8262e7bf05b69d4921d52cda987b17 includes the changes. Can you check they're all good?

Thanks so much!

**Comment by purplesyringa** [comment]
Check my new PR (#4)

**Pull request merged**